tox.ini

```diff
@@ -0,0 +1,16 @@
+[tox]
+envlist = py{36,37,38,39,310,311}
+skip_missing_interpreters = true
+skipsdist = true
+isolated_build = true
+
+[testenv]
+recreate = true
+usedevelop = true
+sitepackages = false
+# pip < 19 requires setup.py
+deps = pip >= 19
+setenv = PYTHONDONTWRITEBYTECODE=1
+commands =
+    python -c "import rpm"
+    python -m pip check
```

Comment:
should we also install the shim module? and then ideally `cd /` to make sure the installation works properly

Reply:
tox does install the shim module before running *commands*, in a virtualenv in a temporary directory (you can run `tox -vv` to see all the details).